packages/lib/computer.d.ts

```diff
@@ -1,11 +1,12 @@
 /// <reference types="node" />
 import {
   Psbt,
-  Transaction,
   TxInput,
   TxOutput,
 } from "@bitcoin-computer/nakamotojs";
 
+import { Transaction } from '@bitcoin-computer/lib'
```

Comment:
We should consider to use Transaction from nakamotoJs and add the extended properties.